### PR TITLE
Add minutes for 2015-05-21

### PR DESCRIPTION
**Pull request by mhdawson**
Adding minutes for last meeting

### CODE REVIEW

File: doc/about/organization/tsc-meetings/2015-05-21/minutes.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+* Alexis Campailla
+* James Snell
+* Julien Gilli
+* Michael Dawson
+
+### Upcoming releases
+
+#### v0.12.4
+Looking to do a 0.12.4 release soon, end of this week or early next week to address
++ breaking v8 change https://github.com/joyent/node/issues/25324 - have PR ready
```
Comment (misterdjules) [inline doc/about/organization/tsc-meetings/2015-05-21/minutes.md]: @mhdawson Unnecessary `+` character here and on the line below.

File: doc/about/organization/tsc-meetings/2015-05-21/minutes.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
++ Windows XP issues, have pull request from io.js, ready to pull over
+
+James also has RC4/Barmizva fix ready to go back in as well
```
Comment (misterdjules) [inline doc/about/organization/tsc-meetings/2015-05-21/minutes.md]: Typo: "Barmizva" -> "Bar Mitzvah".

File: doc/about/organization/tsc-meetings/2015-05-21/minutes.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+### LTS
+
+
+James: Some concern over moving directly to next LTS based on converged repo, we may need
+to allow for a 0.14.X
+
+After discussion we agreed that we need to provide more information so that people have
+a better idea of what will be changing and the impact.  More specifically:
++ gather list of breaking changes that will be in converged repo so that we can document
```
Comment (misterdjules) [inline doc/about/organization/tsc-meetings/2015-05-21/minutes.md]: Unnecessary `+` character. Same a couple lines below.

File: doc/about/organization/tsc-meetings/2015-05-21/minutes.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+and better communicate what will be different
++ reach out broadly to users to gather input on what their specific concerns might be
```
Comment (misterdjules) [inline doc/about/organization/tsc-meetings/2015-05-21/minutes.md]: Maybe split the sentence here with a dot.

**Comment by misterdjules** [comment]
Thank you @mhdawson once again for taking minutes. In addition to my minor comments, we'll need to add a reference to these minutes from the table of content here: https://nodejs.org/about/organization/tsc-meetings/.

To do that, just add an entry here: https://github.com/joyent/node-website/blob/master/doc/about/organization/tsc-meetings/index.md like following:

```
* 2015-05-21
  - [Minutes](2015-05-21/minutes.html)
```

File: doc/about/organization/tsc-meetings/2015-05-21/minutes.md (hunk 4 above)
Comment (mhdawson) [inline doc/about/organization/tsc-meetings/2015-05-21/minutes.md]: The + were actually meant to make those two lines separate bullet points. And I think they looked ok under the gist markdown editor.  In any case   Added CRLF after "More specifically:" and then changed + to \*  and re-validated that this shows up as a bullet list in the markdown view

**Comment by mhdawson** [comment]
Updated as per comments.  Maybe next time I'll remember to add the link in the index the first time :)

**Comment by misterdjules** [comment]
LGTM. Thanks @mhdawson :+1:

**Comment by misterdjules** [comment]
Landed in 873ce039911dc63b6f0f3e8998f05e8ec23dcf19 and deployed at https://nodejs.org/about/organization/tsc-meetings/2015-05-21/minutes.html.

**Pull request closed without merging**